packages/trpc-interface/src/shared/cms.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Localhost implementation of the dashboard trpc interface
```

Comment:
cms name is very confusing here, dashboard isn't cms

Reply:
I think what you want is a "dashboard" service

Reply:
Wrong comment, see below